web/src/routes/Birdseye.jsx

```diff
@@ -80,23 +83,30 @@ export default function Birdseye() {
         {config.birdseye.restream && (
           <select
             className="basis-1/8 cursor-pointer rounded dark:bg-slate-800"
-            value={viewSource}
-            onChange={(e) => setViewSource(e.target.value)}
+            value={isMaxWidth ? "true" : "false"}
+            onChange={(e) => setIsMaxWidth(e.target.value)}
```

Comment:
This seems like it should not have changed, now there is no way to set the source of the live view?